### ISSUE TITLE
Dantooine Ruins - Sequence break in Star Map reveal cutscene

### ISSUE THREAD

**Issue by ghost** [KOTORCommunityPatches/K1_Community_Patch#590]
It is possible to get locked out of Dantooine's Star Map by clicking on the door before the cutscene starts, which prevents the cutscene from firing and the Star Map from opening.

**Comment by DarthParametric** [comment]
Which door? The entrance to the Star Map room?

**Comment by LaIngobernable** [comment]
Yes, the one right before the Star Map. I was rushing through that part to check something else and managed to lock myself out of the map, as it never opened.

**Comment by DarthParametric** [comment]
So multiple clicks on the door then? I suppose it might be possible to break it that way. I'll need to test it, but presumably it could be prevented with the addition of a `NoClicksFor`.

**Comment by LaIngobernable** [comment]
I think I was just pressing W and R at the same time. I may have pressed R more than once.

**Comment by LaIngobernable** [comment]
Tried to actively break the sequence just now and wasn't able to recreate it. In fact, you need to click on the door for the cutscene to start. Might have been a fluke.

**Comment by DarthParametric** [comment]
There's already a `NoClicksFor` in the door's OnOpen script that starts the scene ([k_pdan_cut51](https://github.com/KOTORCommunityPatches/K1_Community_Patch/blob/master/Source/k_pdan_cut51.nss)), but it doesn't fire until after the fadeout, so I guess it could stand to get shuffled up to the top of the sequence.

**Comment by DarthParametric** [comment]
Seems like this is still potentially a problem - https://deadlystream.com/topic/11634-kotor-1-stuck-before-dantooine-star-map

**Comment by DarthParametric** [comment]
This latest incarnation actually seems to be a different issue from the original one. The original one couldn't open the door, the latest one appears to have quick saved right as they opened the door and then loaded that save, leaving them with an open door and no way to trigger the cutscene.

Seems like it might be prudent to add an autosave at the end of the script where the barrier is removed from the previous door (`dan15_opendoor`) before the player gets a chance to touch the Star Map door.
